canonical-web-and-design/ubuntu.com
# Add mir engage page

## Done

- Add 'build-smart-devices-with-mir-whitepaper` engage page
- Add whitepaper card to `/engage`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/build-smart-devices-with-mir-whitepaper
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1181)
- Compare to [copy](https://docs.google.com/document/d/11E9O_mVN_U63ZLsZOvwsdXAyNtpRv0aF8_VWBDfrnRc/edit#)

P.S. PDF is not ready yet


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1182
